.github/workflows/appeal-reply-service-api.yml

```diff
@@ -0,0 +1,110 @@
+name: Appeal Reply Service API
+
+env:
+  APP_DIR: appeals-reply-service-api
```

Comment:
Change all instances of `appeals-reply-service-api` to `appeal-reply-service-api`

Reply:
Done.